app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
👍 
It actually caused trouble in another PR (to be reopened) as this id has to be unique in a page... I tried to put the the input inside the label (see last commit).